packages/ui-kit/src/blocks/elements/PlainTextInputElement.ts

```diff
@@ -8,4 +8,7 @@ export type PlainTextInputElement = Actionable<{
   multiline?: boolean;
   minLength?: number;
   maxLength?: number;
+  multilineLarge?: boolean;
+  multilineMedium?: boolean;
+  multilineSmall?: boolean;
```

Comment:
is this some part of other protocol? if there is no standard or compatibility I would stay with something like
multilineSize: 'large' | 'medium' | 'small'

Reply:
@ggazzo Yes, seems like a better option to have it that way. We can stick to` multilineSize: 'large' | 'medium' | 'small' ` . I will make the required changes.